CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 ## Header Feature
 
 - Adds static `Header` component with login dropdown (functionality to be added later).
+- Adds @chakra-ui/focus-lock packages to dependencies and peerDependencies.
+- Implements focus lock in `Login` component.
```

Comment:
Say,
```
... in the `Header`'s internal `Login` component.
```
so it's explicit that it's not an exported component but an internal component.